test/xpu_api/utilities/meta/meta.unary/meta.unary.prop/is_pod.pass.cpp

```diff
@@ -0,0 +1,98 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+#include "support/utils_invoke.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+#    if TEST_STD_VER == 17
```

Comment:
I am not sure it is correct to completely disable these test for C++20 and above, because this functionality was not yet removed from the STL and oneDPL. I think we need to test it and may be suppress some warnings (if any). If there are no warnings in this test in C++20 mode, this comment can be closed and ignored.

Reply:
Fixed (reverted)